projects/sdk-wells/src/lib/WellFunction.ts

```diff
@@ -0,0 +1,3 @@
+export class WellFunction {
+  constructor(public readonly address: string, public readonly data: string) {}
```

Comment:
The IWellFunction also has a `name` property, can we surface it here?

Reply:
Was going to do a separate PR for these (well function, pump, auger, etc..)